packages/ui-extensions/src/surfaces/customer-account/components/Page/Page.ts

```diff
@@ -0,0 +1,42 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {RemoteFragment} from '@remote-ui/core';
+
+export interface PageProps {
+  /**
+   * The text to be used as title.
+   */
+  title: string;
+
+  /**
+   * The text to be used as subtitle.
+   */
+  subtitle?: string;
+
+  /**
+   * The action grouping, provided as button(s), that is placed in the primary position of the page.
+   */
+  primaryAction?: RemoteFragment;
+
+  /**
+   * Label for the primary action grouping.
+   *
+   * @defaultValue "More actions"
+   */
+  primaryActionLabel?: string;
```

Comment:
You'll also want to add the [primaryActionAccessibilityLabel](https://github.com/Shopify/ui-api-design/pull/152/files#diff-a9ce6ebb5f6c0df35b9ce43a1137666ea51d748c3c3e6aa80b3f565b7b1138a9R25)